### ISSUE TITLE
Topic 7: Notes and Note Type

### ISSUE THREAD

**Issue by rondlg** [fieldmuseum/EMu-Documentation#1091]
DISCUSSION POINTS:Lots of random stuff in the field. Need to decide on appropriate values lookup Transactions Note Type.

**Comment by rondlg** [comment]
DECISION(S): Switch off Old Tab for all except Admin. For both accession and loan types.
DATA CHANGES: 
[Data Changes Doc](https://docs.google.com/spreadsheets/d/1ooFIbkycegoM2cqEZuMn8Ls8l_y5P-axptpAE7qKW3I/edit#gid=0)
Pre new development: row 3,  row 4
Post development

**Comment by rondlg** [comment]
- AllNotTab no longer active for groups other than Admin

- @fmjjones and @magpiedin Notes have been transferred to the MV notes tab can we clear out the NotNotes field?

**Comment by magpiedin** [comment]
Yes -- @fmjjones -- want me to clear those?  (I saved a group of ~23K records - "Standards - clear NotNotes")

**Comment by fmjjones** [comment]
Yes to both.

**Comment by rondlg** [comment]
@magpiedin If this was cleaned up go ahead and close out the ticket
